snake/snakeBody.py

```diff
@@ -0,0 +1,58 @@
+from enum import Enum
+
+class Direction(Enum):
+    NORTH = 1
+    SOUTH = 2
+    EAST = 3
+    WEST = 4
+
+class SnakeBody:
+    x = 0
+    y = 0
+    size = 0
+    direction = Direction.NORTH
+    bodyParts = []
+
+    def __init__(self, posX, posY, snakeDir, parts):
+        self.x = int(posX)
+        self.y = int(posY)
+        self.direction = snakeDir
+        self.bodyParts = parts
```

Comment:
So for a saved game, we have to parse the map and create a list for with the coordinate of each body parts ?

Reply:
Yes, however I can create a function in the GameMap class that will do when it receives a map as parameter. I will do it in another issue.